### PR TITLE
Fix #304 manual restart fails

### LINKED ISSUE #304: Manual round restart fails, verifier uploads response during the round reset
Currently the verifiers first upload their verifications `/v1/upload/challenge/...` and then `try_verify` `/v1/verifier/try_verify/...` to perform verification. Verifiers seem to be able to upload verifications even during the round reset ( #288 ) process, even though they have technically already been removed from the round (but not in the cached state in the operator in `aleo-setup-coordinator` seemingly). This is probably a serious problem, and seems to be causing problems in the restarted round when these files are unexpectedly already present. This behaviour may also be present with contributors, and could also explain some of the problems encountered with implementing the replacement contributors.

This behaviour can be seen on line 50570 in the log:
[coordinator.log](https://github.com/AleoHQ/aleo-setup/files/6811172/coordinator.log)

```txt
Jul 11 17:02:28.956 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removed file ./transcript/development/round_1/chunk_36/contribution_1.unverified
Jul 11 17:02:28.957 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removed ./transcript/development/round_1/chunk_36/contribution_1.unverified
Jul 11 17:02:28.957 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removing file ./transcript/development/round_1/chunk_36/contribution_1.unverified.signature
Jul 11 17:02:28.957 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removed file ./transcript/development/round_1/chunk_36/contribution_1.unverified.signature
Jul 11 17:02:28.957 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removed ./transcript/development/round_1/chunk_36/contribution_1.unverified.signature
Jul 11 17:02:28.957 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removing file ./transcript/development/round_1/chunk_36/contribution_1.verified
Jul 11 17:02:28.957 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removed file ./transcript/development/round_1/chunk_36/contribution_1.verified
Jul 11 17:02:28.958 TRACE aleo_setup_coordinator::api::upload::challenge: Response uploaded from: aleo1em2sc2e24f6kg6ksr690etmz6tnp7h4tjekedwnhukvqy2snkuxs4lw67p.verifier to transcript/development/round_2/chunk_26/contribution_0.verified
Jul 11 17:02:28.958 TRACE aleo_setup_coordinator::api::upload::challenge: Contribution file signature uploaded from: aleo1em2sc2e24f6kg6ksr690etmz6tnp7h4tjekedwnhukvqy2snkuxs4lw67p.verifier to transcript/development/round_2/chunk_26/contribution_0.verified.signature
Jul 11 17:02:28.958 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removed ./transcript/development/round_1/chunk_36/contribution_1.verified
Jul 11 17:02:28.958 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removing file ./transcript/development/round_1/chunk_36/contribution_1.verified.signature
Jul 11 17:02:28.958 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removed file ./transcript/development/round_1/chunk_36/contribution_1.verified.signature
Jul 11 17:02:28.959 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removed ./transcript/development/round_1/chunk_36/contribution_1.verified.signature
Jul 11 17:02:28.959 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removing file ./transcript/development/round_1/chunk_36/contribution_2.unverified
Jul 11 17:02:28.959 TRACE reset_round{round=1}: phase1_coordinator::storage::disk: Removed file ./transcript/development/round_1/chunk_36/contribution_2.unverified
```

another upload that now fails because the participant has now been removed. I'm not sure why there isn't a `try_verify` between these two, or perhaps one of these log lines is from `try_verify` and the other is from the other verifier... It warrants more investigation.

```txt
Jul 11 17:02:29.096 TRACE phase1_coordinator::storage::disk: Fetched ./transcript/development/round_0/state.json
Jul 11 17:02:29.096 ERROR aleo_setup_coordinator::operator: Participant aleo1em2sc2e24f6kg6ksr690etmz6tnp7h4tjekedwnhukvqy2snkuxs4lw67p.verifier is not authorized
Jul 11 17:02:29.096 ERROR phase1_coordinator::coordinator: ParticipantUnauthorized
Jul 11 17:02:29.096 ERROR phase1_coordinator::coordinator: ParticipantUnauthorized
Jul 11 17:02:29.096 ERROR aleo_setup_coordinator::api::upload::challenge: Error uploading next challenge file ParticipantUnauthorized
Jul 11 17:02:29.657 TRACE aleo_setup_coordinator::authentication::authentication_middleware: Authentication for address aleo15fc08f4pcky20f63ldqa97guc4gr554w7zduxmt7yuf3dv7dacxqpflffg message is: "post /v1/verifier/try_lock"
```

### PR DESCRIPTION
**Pull request by kellpossible**
Fix for #304 . Changes based on top of #311 

+ Remove contributions/verification files that were created when locks were
  taken and the locators initialized.
+ Some more tracing instrumentation.
+ Remove some unnecessary inlines.
+ Better logging for timeout drops.

### CODE REVIEW

**Review by julesdesmit — APPROVED** [review]

File: Cargo.lock [hunk 1]
```diff
@@ -2503,6 +2512,7 @@ dependencies = [
  "algebra",
  "anyhow",
  "chrono",
+ "chrono-humanize",
```
Comment (ibaryshnikov) [inline Cargo.lock:2515]: Do we really need an extra crate to print the number of minutes/seconds? `chrono` itself is already a good additional to std types.

Comment (kellpossible) [reply Cargo.lock:2515]: it's easier for operators to read, but I'm willing to remove it if you think the cost is too high

File: phase1-coordinator/src/coordinator.rs [hunk 2]
```diff
@@ -1447,7 +1458,6 @@ where
     ///
     /// On failure, this function returns a `CoordinatorError`.
     ///
-    #[inline]
```
Comment (ibaryshnikov) [inline phase1-coordinator/src/coordinator.rs:1450]: This line has nothing to do with round restart. It'll be nice to revisit our inline policy and to submit a separate PR deleting all inlines which are not justified by the benchmarks. Inlines are not always good for performance, but they always increase the compile times. @apruden2008 do you think it worth the time doing something with inlines?

File: Cargo.lock (hunk 1 above)
Comment (ibaryshnikov) [reply Cargo.lock:2515]: My concern here is not about compile times, but about readability. It's not a blocker for merging this PR though.

When it comes to reasoning about the code, the behaviour of
```rust
println!("{}m {}s", duration.num_minutes(), duration.num_seconds());
```
is more obvious than
```rust
println!("{}", HumanTime::from(duration));
```

File: phase1-coordinator/src/coordinator_state.rs [hunk 3]
```diff
@@ -2747,16 +2747,26 @@ impl CoordinatorState {
...
+                if !self.is_coordinator_contributor(&participant) && !exceeded_chunk_names.is_empty() {
+                    let exceeded_chunks_string: String = exceeded_chunk_names.join(", ");
+
+                    tracing::warn!(
+                        "Dropping participant {} because it has exceeded the maximum ({}) allowed time \
+                        it is allowed to hold a lock (on chunks {}).",
+                        participant,
+                        chrono_humanize::HumanTime::from(participant_lock_timeout),
+                        exceeded_chunks_string,
+                    );
```
Comment (ibaryshnikov) [inline phase1-coordinator/src/coordinator_state.rs:2769]: @kellpossible this part has the same logic, but better reporting on what chunks caused the drop, right?

**Review by ibaryshnikov — APPROVED** [review]

**Review by apruden2008 — APPROVED** [review]

File: phase1-coordinator/src/coordinator.rs (hunk 2 above)
Comment (kellpossible) [reply phase1-coordinator/src/coordinator.rs:1450]: And sometimes they even decrease performance! A number of posts about that on the Rust Reddit I've seen recently.

File: phase1-coordinator/src/coordinator_state.rs (hunk 3 above)
Comment (kellpossible) [reply phase1-coordinator/src/coordinator_state.rs:2769]: @ibaryshnikov yes that was my intention, to report exactly which chunk locks had their limit exceeded.
